src/backend/marsha/core/serializers.py

```diff
@@ -299,6 +302,31 @@ class Meta:  # noqa
     thumbnail = ThumbnailSerializer(read_only=True, allow_null=True)
     urls = serializers.SerializerMethodField()
     is_ready_to_show = serializers.BooleanField(read_only=True)
+    disabled_use_subtitle_as_transcript = serializers.SerializerMethodField()
+
+    def get_disabled_use_subtitle_as_transcript(self, obj):
```

Comment:
I had some trouble understanding why we need this and what its relation is with the manual toggle, until I realized it's just there to let the frontend know _"there are available transcripts so do not fallback to subtitles"_, without having to make an additional request.

IMO this would be much clearer if it were called `has_available_transcripts` or something to that effect

Reply:
or just `has_transcript` ? What information does `available` add?